wolox-training/igc-rails
# Versionist

### Summary
- Added structure for API using versionist
- Added update method for API
- Added inline edit to article listing

### Trello card
https://trello.com/c/T5E0Z60j/24-crear-update-de-articulos-con-javascript

### Test plan
1. Go to the article listing
2. Click on inline edit, a form should appear
3. Change the values and submit the form.
4. The row for the clicked article should be updated with the new values now.
5. Reload the listing
6. The article should have been updated.
